scenarios/local/podman/vm.go

```diff
@@ -0,0 +1,40 @@
+package localpodman
+
+import (
+	"github.com/DataDog/test-infra-definitions/components"
+	"github.com/DataDog/test-infra-definitions/components/command"
+	componentsos "github.com/DataDog/test-infra-definitions/components/os"
+	"github.com/DataDog/test-infra-definitions/components/remote"
+	"github.com/DataDog/test-infra-definitions/resources/local"
+	localpodman "github.com/DataDog/test-infra-definitions/resources/local/podman"
+
+	"github.com/pulumi/pulumi/sdk/v3/go/pulumi"
+)
+
+// NewVM creates an localpodman Ubuntu VM Instance and returns a Remote component.
+// Without any parameter it creates an Ubuntu VM on AMD64 architecture.
```

Comment:
Maybe we could make it more clear that it does not really create a VM but rather a container that is supposed to behave like a VM.
The second line is also a bit strange because we cannot pass any parameter. We can just indicate that it creates a Ubuntu container with AMD64 architecture